### PR TITLE
Added colorFuzz to MagickImage

### PR DESCRIPTION
**Pull request by Peeterush**
I would love to use this in one of my own projects, but not all functionality is there yet. So I figured maybe I can help a bit :)

Let me know if I understood things wrongly!

### CODE REVIEW

File: src/percentage.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ export class Percentage {
         this._value = value;
     }
 
+    static fromQuantum(value: number): Percentage {
```
Comment (dlemstra) [inline src/percentage.ts:13]: Can you mark this method as internal? (`/** @internal */`)

**Review by dlemstra — COMMENTED** [review]
Thanks so much for your pull requests. And thanks for following the coding style of the project. Just a minor comment.

**Review by dlemstra — APPROVED** [review]
Thanks for helping me. I can publish a new release that includes this if that would help you.

**Comment by Peeterush** [comment]
I think there's more functions I need before I can do all I want, so no need to publish for me yet. If I have more free time the coming weeks I might make more pull requests!

**Comment by dlemstra** [comment]
Thanks for offering to help me out!

**Pull request merged**